rpc/jsonrpc/server/http_server_test.go

```diff
@@ -85,12 +81,18 @@ func TestServeTLS(t *testing.T) {
 		fmt.Fprint(w, "some body")
 	})
 
+	chErr := make(chan error, 1)
 	go func() {
-		if err := ServeTLS(ln, mux, "test.crt", "test.key", log.TestingLogger(), DefaultConfig()); err != nil {
-			t.Log(err)
```

Comment:
t.Log is safe

Reply:
Interesting, the race detector was throwing errors because of this line.

Reply:
I reran the job so I'm unfortunately not able to get the build output, but will keep this in mind for the next time something similar happens. I may just have misread the goroutine dump or something.